sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -741,6 +752,13 @@ public synchronized void shutdown() {
         suspend(false, new CancellationException());
     }
 
+    @Override
+    public void onNewAuthToken(String authToken) {
```

Comment:
double check threading here and possibly add synchronized if needed

Reply:
I don't think it needs to be synchronized unless `mIngestion` can disappear on another thread somehow

Reply:
Which may be possible, I'm not sure